cli/cmd/proxy/read/command_test.go

```diff
@@ -53,9 +56,11 @@ func TestFlagParsing(t *testing.T) {
 }
 
 func TestReadCommandOutput(t *testing.T) {
```

Comment:
Should we add happy path cases for Raw and JSON output cases? And maybe we could consider adding a happy path multiport case, maybe just with one of the types of output? Then this file could kind of cover some end to end happy path cases, and format_test.go is covering the table formatting cases.

Reply:
Yes, are you ok if I make a note of it and come back around to add the tests?